tools/individual-packages-gen/gen-phase1.ts

```diff
@@ -0,0 +1,209 @@
+          for (const dependency of Object.keys(srcPackageJson.dependencies || {})) {
+            // all 'regular' dependencies on alpha modules will be converted to
+            // a pair of devDependency on '0.0.0' and peerDependency on '^0.0.0',
+            // and the package will have no regular dependencies anymore
+            if (alphaPackages[dependency]) {
+              alphaDependencies[alphaPackages[dependency]] = pkg.version;
+            }
+          }
+          srcPackageJson.dependencies = undefined;
+
+          // devDependencies
+          const alphaDevDependencies: { [dep: string]: string } = {
+            // we need jsii-rosetta in the dependencies,
+            // otherwise the binary cannot be found when running the 'extract' script
+            'jsii-rosetta': jsiiRosettaVersion,
+          };
+          const devDependencies = srcPackageJson.devDependencies || {};
+          for (const devDependency of Object.keys(devDependencies)) {
+            if (devDependency.startsWith('@aws-cdk/')) {
+              delete devDependencies[devDependency];
+            }
+            if (alphaPackages[devDependency]) {
+              alphaDevDependencies[alphaPackages[devDependency]] = pkg.version;
+            }
+          }
+          devDependencies['@aws-cdk/assert'] = pkg.version;
+          devDependencies['aws-cdk-lib'] = pkg.version;
+          devDependencies.constructs = '^10.0.0';
+          // we save the devDependencies in a temporary key in package.json
+          srcPackageJson.tmp_devDependencies = {
+            ...devDependencies,
+            ...alphaDevDependencies,
+            ...alphaDependencies,
+          };
+          srcPackageJson.devDependencies = {
+            ...alphaDevDependencies,
+            ...alphaDependencies,
+          };
+
+          // peer dependencies
+          srcPackageJson.tmp_peerDependencies = {
+            'aws-cdk-lib': `^${pkg.version}`,
+            'constructs': '^10.0.0',
+            ...(Object.keys(alphaDependencies)
+              .reduce((acc, alphaDependency) => {
+                acc[alphaDependency] = `^${pkg.version}`;
+                return acc;
+              }, {} as { [dep: string]: string })),
+          };
+          srcPackageJson.peerDependencies = undefined;
+
+          // turn off the L1 generation, which uses @aws-cdk/ modules
+          delete srcPackageJson.scripts.gen;
+
+          fileProcessed = true;
+          fs.writeJsonSync(destination, srcPackageJson, { spaces: 2 });
+        }
+      } else if (sourceFileName === '.gitignore') {
+        // ignore everything, otherwise there are uncommitted files present in testing,
+        // because the module's .gitignore file has entries like !.eslintrc.js
+        fileProcessed = true;
+        const gitIgnoreContents = fs.readFileSync(source);
+        fs.outputFileSync(destination, Buffer.concat([gitIgnoreContents, Buffer.from('\n*\n')]));
```

Comment:
Minor: Maybe simpler to add a single git ignore file at `individual-package/.gitignore`?

Reply:
As a comment in a different part of this file says, `pkglint` checks for `.gitignore`.